resources/build/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN git clone --branch v42 https://github.com/rollbear/trompeloeil
 RUN cd trompeloeil && cmake -S . -B build && cmake --build build && cmake --install build
 
 # Pull and install fmt
-RUN git clone --branch 9.1.0 https://github.com/fmtlib/fmt
+RUN git clone --branch 10.2.1 https://github.com/fmtlib/fmt
```

Comment:
I don't know how to do ... any of that.

The ABI dump is using the `v9` inline namespace, which indicates that it's still using version 9, which is surprising.
Do you have a reference to where you originally discovered the forced export symbol, the original comment you left dosen't provide any guidance. When we were pairing you said that now version 10 dosen't export it so we should update, but I have no idea where or how you learned that.

Given that we changed this during development as a hail mary solve, can we just drop this commit? It's not neccesary and not part of any AC's related to the work done here. Could just plan to do it independently if it ever becomes neccesary ... which it may not for a long while.

Reply:
I'm happy to drop the commit.

I think it was fixed in version 10 of the library, which might not correspond to the inline namespace(?)

You're right that we don't have any process instructions for updating the Docker image.  Usually it's obvious its needed because tests fail until you update, but obviously not the case here.

I had in mind to reset back to the commit that bumped fmtlib and run the ABI check, see if it fails, and if so why. The README details running the ABI check locally as one of the final steps.  But perhaps that should be moved to before the snapshot is updated, to reproduce the ABI check failure locally first before moving on to update the snapshots.

Reply:
Aye. I just dropped the commit on this.